web/src/components/Meditor/Meditor.vue

```diff
@@ -331,6 +381,29 @@ export default defineComponent({
       (p) => renderField((complexSchema as any).properties[p]),
     );
 
+    function loadDataFromLocalStorage() {
+      // load previous meditor data from localStorage
+      editorInterface.value.setModel(getModelLocalStorage(model.value!.id));
```

Comment:
I know this is never intended to be called when `model.value` is `undefined`, but to easily guard against it, at the top of the file you could just write

```typescript
      if (!model.value) {
        return;
      }
```

I don't feel too strongly I suppose, but this would potentially prevent errors from being thrown. Do whatever you think is best.

Reply:
https://github.com/dandi/dandi-archive/pull/947/commits/a495f79f99d03c8b03d22462176af51f5b31dc02